security/pkg/nodeagent/test/cert_rotation/rotation_test.go

```diff
@@ -25,7 +25,7 @@ import (
 )
 
 const (
-	rotateInterval   = 1500 * time.Millisecond
+	rotateInterval   = 2000 * time.Millisecond
```

Comment:
This rotateInterval sets the frequency of cert rotation at SDS agent. Making it longer does not deflake the test. We could leave it unchanged and increase proxyRunningTime.